sjb/config/common/test_cases/origin.yml

```diff
@@ -25,6 +25,7 @@ provision:
 sync_repos:
   - name: "origin"
   - name: "image-registry"
+  - name: "online-hibernation"
```

Comment:
Err -- sorry -- you should add this to the overrides list in your jobs, but we should not ask all Origin jobs to clone down this repo!